### DISCUSSION TITLE
полет по определенной высоте из API

### DISCUSSION THREAD

**Discussion by Kukushechka7** [Sokol-Games-GmbH/PixelCombats.PublicApi#26] [Ideas]
1. Сделать метод SetFlyh();
Описание: 
Поднимает игрока на определённую высоту
Пример: 
`player.Build.SetFlyh(100);`

**Comment by kkohno** [comment]
ну тут не делали просто по той причине чтобы игрок не улетал люто вверх или вниз и не терялся там)) хотябы както ограничить это... просто можно улететь так, что у Вас там глючить начнет))

**Comment by Kukushechka7** [comment]
Тогда сделайте чтобы игрок падал при отключении флая

**Comment by kkohno** [comment]
обдумаем
